### ISSUE TITLE
Should dates be generated within surrounding quotes?

### ISSUE THREAD

**Issue by paulrobertlloyd** [11ty/eleventy#258]
Given the following front matter:

```yaml
class: classname
date: 2009-04-15T01:34:34+01:00
```

I would expect the following template:

```liquid
<div class="{{ class }}" datetime="{{ date }}">…</div>
```

to generate `<div class="classname" datetime="2009-04-15T00:34:34.000Z">…</div>`. However, for some reason unformatted dates are output with surrounding quotes, thus generating the following invalid HTML:

```html
<div class="{{ class }}" datetime=""2009-04-15T00:34:34.000Z"">…</div>
```

Clearly, the solution is to not use quotes when outputting dates, although this introduces inconsistency into my templates. This becomes especially true should I choose to format a date string. So for example, this template:

```liquid
<div class="{{ class }}" datetime="{{ item.data.date | date: '%Y-%m-%d' }}">…</div>
```

generates: `<div class="classname" datetime="2009-04-15">…</div>` (no double quotes). This behaviour has been gnawing away at me for a while now, so thought best to document it and ask if it could be addressed in a forthcoming update!

Thanks @zachleat! FYI, I'm using Liquid templates; might this be a LiquidJS issue and not an 11ty one?

**Comment by zachleat** [comment]
Yes after a few tests it does look as thought this is coming from Liquid.

<img width="1152" alt="image" src="https://user-images.githubusercontent.com/39355/46250578-f4bff680-c402-11e8-8f08-6e9ae1e10c73.png">

```
test("Liquid Render Date, Issue #258", async t => {
  let fn = await new TemplateRender("liquid").getCompiledTemplate(
    "<p>{{ myDate }}</p>"
  );
  t.is(await fn({ myDate: new Date(Date.UTC(2016, 0, 1, 0, 0, 0)) }), "<p>2016-01-01T00:00:00.000Z</p>");
});
```

I wonder if you could file an issue with https://github.com/harttle/liquidjs and cc the URL here?

**Comment by paulrobertlloyd** [comment]
Ah-ha! Okay, happy to post an issue there… however I note that 11ty isn’t using the latest version of liquidjs (v6.0.0), so perhaps the dependancy should be updated before proceeding?

**Comment by zachleat** [comment]
Ah, after testing it locally the issue is still present in v6.0.0 of `liquidjs`.
